src/patternfly/components/Truncate/truncate-omission.hbs

```diff
@@ -0,0 +1,11 @@
+<span class="{{pfv}}truncate__omission{{#if truncate-omission--modifier}} {{truncate-omission--modifier}}{{/if}}"
+  {{#if truncate-start--attribute}}
+    {{{truncate-start--attribute}}}
+  {{/if}}
+  aria-hidden="true">
```

Comment:
nit - we'll add the `--attribute` bit last to make sure it overrides any previously set attributes.

```suggestion
  aria-hidden="true"
  {{#if truncate-start--attribute}}
    {{{truncate-start--attribute}}}
  {{/if}}>
```